src/System.Text.Json/tests/Serialization/Stream.WriteTests.cs

```diff
@@ -258,6 +263,636 @@ public static async Task LargeJsonFile(int bufferSize)
+                                            OriginalHeight = 556,
+                                            IsFeatured = true,
+                                            FocusPercentX = 30,
+                                            FocusPercentY = 44
+                                        }
+                                    },
+                                    Type = "Tag"
+                                },
+                                new Tag
+                                {
+                                    Slug =  "software-development",
+                                    Name = "Software Development",
+                                    PostCount = 62810,
+                                    Metadata = new Metadata1
+                                    {
+                                        PostCount = 62810,
+                                        CoverImage = new CoverImage
+                                        {
+                                            Id = "0*R1c-uLLQzlqqiA-b",
+                                            OriginalWidth = 8192,
+                                            OriginalHeight = 5461,
+                                            IsFeatured = true,
+                                            UnsplashPhotoId = "UKdCsFKBbkQ"
+                                        }
+                                    },
+                                    Type = "Tag"
+                                }
+                            },
+                            SocialRecommendsCount = i / 2,
+                            ResponsesCreatedCount = 2 + i,
+                            Links = new Links {
+                                Entries = new List<Entry>
+                                {
+                                    new Entry
+                                    {
+                                        Url = new Uri("http://test.com/link/entries/entry/1"),
+                                        Alts = new List<Alt>(),
+                                        HttpStatus = HttpStatusCode.OK
+                                    },
+                                    new Entry
+                                    {
+                                        Url = new Uri("http://test.com/link/entries/entry/2"),
+                                        Alts = new List<Alt>
+                                        {
+                                            new Alt
+                                            {
+                                                Type = 3,
+                                                Url = new Uri("http://test.com/link/entries/alts/1")
+                                            },
+                                            new Alt
+                                            {
+                                                Type = 2,
+                                                Url = new Uri("http://test.com/link/entries/alts/2")
```

Comment:
Artificial URLs should use a host from https://tools.ietf.org/html/rfc2606.

example.com, example.net, example.org, [anything].example, [anything].test, [anything].invalid.

Reply:
will update